deployment/cfn/application.py

```diff
@@ -608,7 +620,14 @@ def get_cloud_config(self, tile_distribution_endpoint):
                 '  - path: /etc/mmw.d/env/MMW_HYDROSHARE_SECRET_KEY\n',
                 '    permissions: 0750\n',
                 '    owner: root:mmw\n',
-                '    content: ', Ref(self.hydroshare_secret_key), '\n',
```

Comment:
Was this line accidentally deleted?

Reply:
No, it was intentional sabotage, but looks like I've been caught. Will add a fixup.